style.css

```diff
@@ -0,0 +1,41 @@
+* {
+	font-family: Verdana, sans-serif;
+	padding-left: 10px;
+	background-color: teal;
+}
+
+nav{
+	text-align: center;
+}
+
+h1{
+	text-align:center;
+}
+
+h2 {
+	display:inline;
+	width:100%;
+	text-align: center;
+	border: solid;
+	margin:20px;
+	background:teal;
+}
+
+footer {
+	text-align: center;
+}
+
+.picf {
+	display: inline;
+}
+
+.social{
+	width:100px;
+	height:100px;
+  	bottom: 0;
```

Comment:
@cliffjadoo this can be omitted, you use this with position `relative` and `absolute` 🔨